tools/cli/commands/build.js

```diff
@@ -831,10 +857,33 @@ async function buildProject( t ) {
 		await fs.writeFile( `${ buildDir }/.gitattributes`, rules, { encoding: 'utf8' } );
 	}
 
+	// Get the proeject version number from the changelog.md file.
+	let projectVersionNumber = '';
+	try {
+		const changelogText = await fs.readFile( `${ t.cwd }/CHANGELOG.md`, { encoding: 'utf8' } );
+		projectVersionNumber = changelogText
+			.toString()
+			.split( '\n' )
+			.find( line => {
+				return line.startsWith( '## ' ) && ! line.includes( '-alpha' );
+			} )
+			.split( ' ' )[ 1 ]
+			.replace( /[[\]]/g, '' );
+
+		if ( ! projectVersionNumber ) {
+			throw 'Version number not found.';
+		}
+	} catch ( err ) {
+		console.error(
+			`Error reading project version number from changelog file for ${ t.project }: ${ err }`
+		);
+		return false;
```

Comment:
This is not how a build failure is signaled.

Reply:
abde726 does this look better?

Reply:
Commented inline.